modules/040-node-manager/crds/sshcredentials.yaml

```diff
@@ -44,6 +44,10 @@ spec:
                   description: |
                     Private SSH key in PEM format encoded as base64 string.
                   type: string
+                privateSSHKeyPassphrase:
```

Comment:
This field can be specified only if ssh keys are provided, but there is no validation